DBTableToCDAPTable/README.rst

```diff
@@ -41,7 +51,7 @@ Configurations for the Database Table Source
 
 #. ``splitBy``: The is the field used to generate splits. Not required if numSplits is set to one.
 
-#. ``numSplits``: The number of splits to generate.
+#. ``numSplits``: A string of the number of splits to generate (optional).
```

Comment:
Added as https://issues.cask.co/browse/CDAP-7180. Will remove from here to complete this PR.